index.html

```diff
@@ -44,7 +44,7 @@
   </head>
   <body>
 
-    <h1>No</h1>
+    <h1>Yes</h1>
     <script>
       // beware of leading zeroes, they indicate octal.
       countup(2016,6,23)
```

Comment:
Agreed. I'm going to simplify for today, and come back to countup later. 

Thanks, all.